src/App.js

```diff
@@ -59,10 +61,18 @@ const App = () => {
           onChange={(e) => setTermsConditions(e.target.checked)}
         />
         <div className="btn-wrap">
-          <div className="btn" onClick={(e) => handleSubmit()} id="btn-submit" aria-label="arial-btn"> Save Data </div>
+          <div
+            className="btn"
+            onClick={(e) => handleSubmit()}
```

Comment:
onClick={(e) => handleSubmit()} can be written as onClick={handleSubmit}. no need of e if you are not passing in function